frontends/web/src/routes/device/settings/components/hiddenwallet.jsx

```diff
@@ -127,6 +129,7 @@ export default class HiddenWallet extends Component {
                                     onValidPassword={this.setValidPIN} />
                                 <PasswordRepeatInput
                                     idPrefix="password"
+                                    pattern="^.{8,}$"
```

Comment:
The pattern attribute is nice, but if it is always active you can't submit the form in this case you never see the error. A user would not be informed that the password is too short when the input field only gets red.

Reply:
Makes sense. Should I remove the pattern, or add a small text `min. 8 characters`?

Reply:
yes please remove the attribute for now

Reply:
We could add it later in another PR, after we are sure there was an error, but maybe have to allow submitting even if it is invalid.